src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -145,7 +146,7 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
      */
     @Advanced
     @Argument(fullName = "input_prior", shortName = "inputPrior", doc = "Input prior for calls", optional = true)
-    public List<Double> inputPrior = Collections.emptyList();
+    public List<Double> inputPrior = new ArrayList<>();
```

Comment:
is this because our CLI adds to the list rather than replacing it? If yes, can you do an audit (or enter a ticket for an audit) of all of our `@Argument` fields that are immutable collections like `Collections.emptyList()` ?